mail/admin.py

```diff
@@ -0,0 +1,25 @@
+"""
+Admin views for Mail app
+"""
+
+from django.contrib import admin
+
+from mail.models import FinancialAidEmailAudit
+
+
+class FinancialAidEmailAuditAdmin(admin.ModelAdmin):
```

Comment:
I'm seeing an option to save the audit, which changes the modified time. If `has_change_permission` was implemented and returned `False`, would that disable the ability to save in the admin?